requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 --requirement requirements-test.txt
 ipython
+mypy
```

Comment:
What are you using `mypy` for?

Reply:
@dav3r This one was me. It was added in https://github.com/cisagov/skeleton-python-library/pull/65 at my suggestion.

Reply:
LOL- I forgot all about that! 🤦